tests/default/data/sql-ast-narrowing.php

```diff
@@ -365,4 +365,21 @@ public function joinSelectOutsideCondition(PDO $pdo): void
         assertType('PDOStatement<array{adaid: int<-32768, 32767>, 0: int<-32768, 32767>, eadavk: numeric-string|null, 1: numeric-string|null}>', $stmt);
     }
 
+    public function multipleJoins(PDO $pdo): void
+    {
+        $stmt = $pdo->query('SELECT adaid, eladaid, c_int, c_char5 from ada inner join ak on adaid = eladaid inner join typemix on adaid = c_int');
+        assertType('PDOStatement<array{adaid: int<-32768, 32767>, 0: int<-32768, 32767>, eladaid: int<-32768, 32767>, 1: int<-32768, 32767>, c_int: int<-32768, 32767>, 2: int<-32768, 32767>, c_char5: string, 3: string}>', $stmt);
+
+        $stmt = $pdo->query('SELECT adaid, eladaid, c_int, c_char5 from ada left join ak on adaid = eladaid left join typemix on adaid = c_int');
+        assertType('PDOStatement<array{adaid: int<-32768, 32767>, 0: int<-32768, 32767>, eladaid: int<-32768, 32767>|null, 1: int<-32768, 32767>|null, c_int: int<-32768, 32767>|null, 2: int<-32768, 32767>|null, c_char5: string|null, 3: string|null}>', $stmt);
+    }
+
+    public function ignoredAstQueries(PDO $pdo): void
+    {
+        $pdo->query('SELECT adaid from ada inner join (select akid from ak) on akid = adaid');
```

Comment:
because it has a different output for `pdo-mysql` and `mysqli` reflectors (but that is unrelated to AST analysis since we give up on that due to the subquery)

Reply:
without an assertion this line does not have value. what is the difference between the reflectors regarding this statement?

Reply:
my bad, must have been doing something wrong back then when i was testing. I just pushed a new assertion, hopefully CI will be green